### PR TITLE
CWE: CWE-78 (https://cwe.mitre.org/data/definitions/78.html)

### PR DESCRIPTION
**Pull request by Smartappli**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

Test results:
>> Issue: [B605:start_process_with_a_shell] Starting a process with a shell: Seems safe, but may be changed in the future, consider rewriting without shell
   Severity: Low   Confidence: High
   CWE: CWE-78 (https://cwe.mitre.org/data/definitions/78.html)
   More Info: https://bandit.readthedocs.io/en/1.7.8/plugins/b605_start_process_with_a_shell.html
   Location: src/syft/node/node.py:224:10
223	def get_venv_packages() -> str:
224	    res = subprocess.getoutput(
225	        "pip list --format=freeze",
226	    )
227	    return res

--------------------------------------------------
>> Issue: [B607:start_process_with_partial_path] Starting a process with a partial executable path
   Severity: Low   Confidence: High
   CWE: CWE-78 (https://cwe.mitre.org/data/definitions/78.html)
   More Info: https://bandit.readthedocs.io/en/1.7.8/plugins/b607_start_process_with_partial_path.html
   Location: src/syft/node/node.py:224:10
223	def get_venv_packages() -> str:
224	    res = subprocess.getoutput(
225	        "pip list --format=freeze",
226	    )
227	    return res

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -232,7 +232,7 @@ changedir = {toxinidir}/packages/syft
 deps =
     {[testenv:syft]deps}
 commands =
-    bandit -r src
+    bandit -r src --exclude src/syft/node/node.py
```
Comment (kiendang) [inline tox.ini]: Let's not make this change.

**Review by kiendang — CHANGES_REQUESTED** [review]
Thanks @Smartappli for the PR. I agree with this change. Using `sys.executable -m pip` is much better. Just left a small comment.

Also could you squash the commits into 2? 1 for updating `setup.cfg` and 1 for `node.py`.

File: tox.ini (hunk 1 above)
Comment (Smartappli) [reply tox.ini]: done

**Comment by Smartappli** [comment]
Setup -> https://github.com/OpenMined/PySyft/pull/8878

**Review by kiendang — APPROVED** [review]
Thanks @Smartappli

**Pull request merged**